studio-client/package.json

```diff
@@ -3,17 +3,17 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": "16",
-    "pnpm": "^7.1.5"
+    "node": "18",
```

Comment:
same questions as in build.yml

Reply:
node is used as a build tool. we will go with the newer version